app/src/main/java/com/example/photogallery/Utils/FileHelper.java

```diff
@@ -0,0 +1,20 @@
+package com.example.photogallery.Utils;
+
+import android.app.Activity;
+import android.os.Environment;
+
+import java.io.File;
+import java.io.IOException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class FileHelper implements IFileHelper {
+    @Override
+    public File createFile(Activity context) throws IOException {
+        String timeStamp = new SimpleDateFormat("yyyyMMdd_HHmmss").format(new Date());
+        String imageFileName = "__" + timeStamp + "_";
+        File storageDir = context.getExternalFilesDir(Environment.DIRECTORY_PICTURES);
+        File image = File.createTempFile(imageFileName, ".jpg", storageDir);
+        return image;
+    }
+}
```

Comment:
Generally speaking, when the file format allows, it is considered best practice to add an empty newline at the end of each new file you introduce.  There are stylistic and functional reasons you can read about [here](https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log) if you are interested.  If we add linting to the project, I expect it will flag this.